### DISCUSSION TITLE
Do databases created at app.supabase.io support realtime subscriptions?

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#1160] [Questions] [answered]
Hi,

Do databases created at app.supabase.io support realtime subscriptions (https://github.com/supabase/realtime) or realtime subscriptions are only supported when the database is setup using the docker image mentioned in the repository README.md?

Thank you in advance for your answer,

Best,

Paulo

**Comment by silentworks** [comment, marked as answer]
This works with databases created at app.supabase.io too. You can see example code in some of the example projects in this repo. The use of the `on()` and `subscribe()` methods is where you will see this happening. You can also read more on it in the documentation https://supabase.io/docs/reference/javascript/subscribe

https://github.com/supabase/supabase/blob/master/examples/nextjs-slack-clone/lib/Store.js - example project code

**Reply by ghost** [reply]
great! thank you.
